src/component.js

```diff
@@ -10,7 +10,7 @@ import {toCSS, destroyElement} from 'belter/src';
 
 function _set_default_props(opts) {
     if (opts.props.color === undefined) {
-        opts.props.color = "#0079ff";
+        opts.props.color = "#216e93";
```

Comment:
Default blue had to be updated, otherwise, it replaces the "pastel-blue" when it's not set in the props of the IFrame.